src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -56,7 +55,7 @@
     <Nullable>enable</Nullable>
 
     <!-- Ignore all previous constants since SPCL is sensitive to what is defined and the Sdk adds some by default -->
-    <DefineConstants>CORECLR;netcoreapp</DefineConstants>
```

Comment:
I assume you verified that nothing is currently depending on `netcoreapp` being defined?

Reply:
Yes.

Reply:
`NETCOREAPP` should be defined regardless provided we are building for some `NETCOREAPP` version: <https://docs.microsoft.com/en-us/dotnet/standard/frameworks#how-to-specify-target-frameworks>

Reply:
No because DefineConstants is overridden here.

Reply:
Ah, right. I completely missed that and the comment above :)